fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingBase.java

```diff
@@ -341,6 +376,49 @@ public void setFallbackMode() {
                 });
     }
 
+    @Nonnull
+    private CompletableFuture<Void> setScrubberTypeOrThrow(FDBRecordStore store) {
```

Comment:
Yes it should. 
I've eliminated the scrubber's type stamp. Please note that: 
1. The type stamp is still defined - as it is required by `IndexerBase`. 
2. I've also eliminated the `indexBuildScannedRecordsSubspace` reset (which is discussed below) because we can't tell now that a scrubber runs for the first time. I suppose it's consistent in the sense that by scrubbing, the scanned records counter grows beyond the indexing process count. 
One thing that may happen is the scrubber successfully reseting its range-sets while an indexer changes the index state to WRITE_ONLY, but I suppose it's harmless now - as the scrubbing's range-sets are separate from the indexing one.